# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kmod/depmod-null-compare.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Don't segfault as os musl 1.2.5

  - NoTag <notag@t2sde.org>

--- kmod-32/tools/depmod.c.vanilla	2024-03-21 11:12:38.054586580 +0100
+++ kmod-32/tools/depmod.c	2024-03-21 11:13:35.989778877 +0100
@@ -3058,7 +3058,7 @@
 	}
 	ctx = NULL; /* owned by depmod */
 
-	if (module_symvers != NULL) {
+	if ((module_symvers == NULL) != 0) {
 		err = depmod_load_symvers(&depmod, module_symvers);
 		if (err < 0) {
 			CRIT("could not load %s: %s\n", module_symvers,
